Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -342,9 +342,9 @@ inline Gfx::ScalingMode to_gfx_scaling_mode(CSS::ImageRendering css_value, Gfx::
             return Gfx::ScalingMode::BoxSampling;
         return Gfx::ScalingMode::BilinearBlend;
     case CSS::ImageRendering::CrispEdges:
-        return Gfx::ScalingMode::NearestNeighbor;
-    case CSS::ImageRendering::Pixelated:
         return Gfx::ScalingMode::SmoothPixels;
+    case CSS::ImageRendering::Pixelated:
+        return Gfx::ScalingMode::NearestNeighbor;
```

Comment:
According to the spec, `Pixelated` allows for minor smoothing (i.e. smooth pixels) and `CrispEdges` does not (i.e. nearest neighbour). The end result is the same though :^)